agents/components/MemberInvites.js

```diff
@@ -81,17 +85,25 @@ function MemberInvites (props) {
   const { styles } = props
   return (
     <form className={styles.container}>
-      <Field
-        name='groupName'
-        type='text'
-        component={TextField}
-        floatingLabelText={
-          <FormattedMessage
-            id='agents.groupName'
-            className={styles.labelText}
-          />
-          }
-      />
+      <p className={styles.intro}>
+        <FormattedMessage
+          id='agents.memberInvites'
+          className={styles.labelText}
+        />
+      </p>
+      <div className={styles.groupNameContainer}>
+        <Field
```

Comment:
hey, i don't think we should have group name as part of the member invites component, this is handled by the group agent profile component, right?